cfme/tests/intelligence/test_dashboard.py

```diff
@@ -66,6 +66,7 @@ def test_widgets_operation(dashboards, widgets, soft_assert, infra_provider):
 
 @pytest.mark.rhel_testing
 @pytest.mark.parametrize("number_dashboards", range(1, 4))
+@pytest.mark.meta(blockers=[BZ(1666712, forced_streams=['5.10'])])
```

Comment:
I don't think forced_streams is necessary here - your fixes to the blocker resolution removes this need! :)

Reply:
Done